src/mam4xx/ndrop.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef MAM4XX_NDROP_HPP
+#define MAM4XX_NDROP_HPP
+
+#include <haero/aero_species.hpp>
+#include <haero/atmosphere.hpp>
+#include <haero/constants.hpp>
+#include <haero/math.hpp>
+
+#include <mam4xx/aero_config.hpp>
+#include <mam4xx/conversions.hpp>
+#include <mam4xx/mam4_types.hpp>
+#include <mam4xx/utils.hpp>
+
+using Real = haero::Real;
+
+namespace mam4 {
+
+KOKKOS_INLINE_FUNCTION
+void get_aer_num(const Diagnostics &diags, const Prognostics &progs,
+                 const Atmosphere &atm, int mode_idx, int icol,
```

Comment:
It seems to me that `icol` is being used for the vertical level index (e.g. `atm.temperature(icol)` is the temperature of the atmospheric column of air at the vertical level `icol`). Our convention for this variable is `k`, whereas `icol` (or `i`) usually refers to which column we're looking at. Can we rename this variable to `k`?